src/main/java/com/ericsson/ei/handlers/ObjectHandler.java

```diff
@@ -136,7 +146,11 @@ public void updateObject(String aggregatedObject, RulesObject rulesObject, Strin
         BasicDBObject document = prepareDocumentForInsertion(id, aggregatedObject);
         String condition = "{\"_id\" : \"" + id + "\"}";
         String documentStr = document.toString();
+        long start = System.currentTimeMillis();
         mongoDbHandler.updateDocument(databaseName, collectionName, condition, documentStr);
+        long end = System.currentTimeMillis();
+        long sec = end - start;
+        LOGGER.info("#### Response time for to update aggregation object with id : {} : {} milliseconds", id, sec);
```

Comment:
I guess this is logged often, change to logger debug.

Reply:
WIP